src/components/formatting/IRSIndicatorLegend/tests/index.test.tsx

```diff
@@ -22,10 +26,15 @@ describe('/components/formatting/IRSIndicatorLegend', () => {
   });
 
   it('renders correctly for namibia indicator colors', () => {
-    const wrapper = mount(<IRSIndicatorLegend indicatorRows={namibiaIndicatorRows} />);
+    const wrapper = mount(
+      <IRSIndicatorLegend
+        indicatorRows={namibiaIndicatorRows}
+        indicatorThresholds={indicatorThresholdsIRSNamibia}
+      />
+    );
     expect(wrapper.props()).toEqual({
       indicatorRows: namibiaIndicatorRows,
-      indicatorThresholds: indicatorThresholdsIRS,
+      indicatorThresholds: indicatorThresholdsIRSNamibia,
```

Comment:
I think that we should add a separate case that deals with Namibia instead of overwriting what was there.